frontends/election-manager/src/components/print_button.tsx

```diff
@@ -66,7 +72,35 @@ export function PrintButton({
       document.title = documentTitle;
     }
 
+    setRenderPrintTarget(false);
     afterPrint?.();
+  }, [
+    afterPrint,
+    afterPrintError,
+    copies,
+    isMounted,
+    makeCancelable,
+    printer,
+    sides,
+    title,
+  ]);
+
+  useEffect(() => {
+    async function printOnTargetRendered() {
+      await print();
+    }
+
+    if (renderPrintTarget) {
+      void printOnTargetRendered();
+    }
```

Comment:
Curious: Why did you have to wrap `print` in `printOnTargetRendered`? I imagine there's a reason why you couldn't just write:

```ts
if (renderPrintTarget) {
  print();
}
```

Reply:
The function passed to `useEffect` [can't be asynchronous](https://devtrium.com/posts/async-functions-useeffect), it turns out, but you can call an asynchronous function within.  I noticed a couple other places in the codebase where the pattern was used, e.g. https://github.com/votingworks/vxsuite/blob/ed7aace62c0bce7afb9e097edd6a00cb325f20dd/frontends/bmd/src/app_root.tsx#L958-L968

Reply:
This has me wondering if I could just do `void print();`... I will see what the linter says.